commonslibrary/settings/base.py

```diff
@@ -56,6 +56,7 @@
     'crispy_forms',
     'django_filters',
     'django_mptt_admin',
+    'import_export',
```

Comment:
Don't think this is used at all as you've gone for a different approach - remove it?